### PR TITLE
corrected some errors in the italian translation

### PR DESCRIPTION
**Pull request by TronFortyTwo**
no less, no more

### CODE REVIEW

**Comment by pharaone** [comment]
there is an error in line 78, "imposibile" should be "impossibile"

**Comment by pharaone** [comment]
another error in line 79, "Autenticazisone" should be "Autenticazione"

File: src/l10n/it.ini [hunk 1]
```diff
@@ -67,17 +67,17 @@ Authfail=Autenticazione fallita
...
 Uploading=Caricamento in corso
 Uploaded=Caricato
 NotUploaded=Errore durante il caricamento
 ErrorUploading=Abbiamo riscontrato un problema durante il caricamento del file. Verifica l'accesso ad internet e riprova.
 DuplicatedUpload=Sembra che tu abbia già caricato questo file.
-NoSDAccess=Impossibile accedere alla scheda di memoria. Verifica che non sia attualmente usata come dispositivo di memoria USB.
-AuthInvalid=Autenticazione errata
-AuthInvalidNotice=L'autenticazione dell'account non è più valida. Si prega di reinserire il numero di telefono per ricevere il codice di conferma via SMS e permettere una valida autenticazione al tuo account.
+NoSDAccess=Imposibile accedere alla scheda di memoria. Verifica che non sia attualmente usata come dispositivo di memoria USB.
+AuthInvalid=Autenticazisone errata
```
Comment (pharaone) [inline src/l10n/it.ini]: should be autenticazione

File: src/l10n/it.ini [hunk 2]
```diff
@@ -67,17 +67,17 @@ Authfail=Autenticazione fallita
...
+DupliAccountNotice=Sei già autenticato con queste credenziali. Riprova con un altro account.
 Uploading=Caricamento in corso
 Uploaded=Caricato
 NotUploaded=Errore durante il caricamento
 ErrorUploading=Abbiamo riscontrato un problema durante il caricamento del file. Verifica l'accesso ad internet e riprova.
 DuplicatedUpload=Sembra che tu abbia già caricato questo file.
-NoSDAccess=Impossibile accedere alla scheda di memoria. Verifica che non sia attualmente usata come dispositivo di memoria USB.
-AuthInvalid=Autenticazione errata
-AuthInvalidNotice=L'autenticazione dell'account non è più valida. Si prega di reinserire il numero di telefono per ricevere il codice di conferma via SMS e permettere una valida autenticazione al tuo account.
+NoSDAccess=Imposibile accedere alla scheda di memoria. Verifica che non sia attualmente usata come dispositivo di memoria USB.
```
Comment (pharaone) [inline src/l10n/it.ini]: should be Impossibile

**Review by pharaone — CHANGES_REQUESTED** [review]
please fix the typo errors

**Comment by TronFortyTwo** [comment]
Ok. Sorry for my low attenction

**Comment by pharaone** [comment]
There is an error in line 70, what's up should be WhatsApp

**Comment by pharaone** [comment]
Nope, in line 70 What's App should be WhatsApp. Senza apostrofo e senza spazio

**Comment by pharaone** [comment]
Same in line 153

**Comment by pharaone** [comment]
👍

**Comment by TronFortyTwo** [comment]
Sorry for all my errors

**Review by pharaone — APPROVED** [review]
All OK

**Pull request merged**